docs/includes/runtimes-note.md

```diff
@@ -1,4 +1,7 @@
 > [!IMPORTANT]
 > For the shared JavaScript runtime, this element enables the ribbon, task pane, and other supported components to use the same runtime. However, the SharedRuntime requirement set is only available in some Office applications. For more information, see [Shared runtime requirement sets](/office/dev/add-ins/reference/requirement-sets/shared-runtime-requirement-sets).
 >
-> **Outlook**: For event-based activation, this element enables your add-in to run on composing a new item, for example. For supported clients and other information, see [Configure your Outlook add-in for event-based activation](/office/dev/add-ins/outlook/autolaunch).
+> **Outlook**
+>
+> - For event-based activation, this element enables your add-in to run on composing a new item, for example. For supported clients and other information, see [Configure your Outlook add-in for event-based activation](/office/dev/add-ins/outlook/autolaunch).
+> - For integrated spam reporting (preview), this element enables your add-in to process unsolicited messages.
```

Comment:
We have a current limitation that the runtime used for LaunchEvents and Spam Reporting MUST be the same runtime. (it is currently unknown if we are going to fix/change this). 

I don't know if htis is the right place to mention it? (maybe in the overal reporting phishing doc, and then link to it from there?)

Reply:
I think the conceptual article would be a better place to add that information.

Reply:
In addition to adding a link to the conceptual article, I've also updated the note based on our discussion.